config/application.rb

```diff
@@ -59,6 +59,41 @@ class Application < Rails::Application
       :omniauthable, { omniauth_providers: %i[saml openid_connect cas] }
     ]
 
+    ##
+    # @!attribute iiif_audio_labels_and_mime_types [r|w]
+    #   @see Hyrax::IiifAv::DisplaysContentDecorator
+    #   @return [Hash<String,String>] Hash of valid audio labels and their mime types.
+    Hyrax::IiifAv::DisplaysContent.class_attribute :iiif_audio_labels_and_mime_types, default: { "ogg" => "audio/ogg", "mp3" => "audio/mpeg" }
+
+    ##
+    # @!attribute iiif_video_labels_and_mime_types [r|w]
+    #   @see Hyrax::IiifAv::DisplaysContentDecorator
+    #   @return [Hash<String,String>] Hash of valid video labels and their mime types.
+    class_attribute :iiif_video_labels_and_mime_types, default: { "mp4" => "video/mpeg", "webm" => "audio/webm" }
```

Comment:
Does this need `Hyrax::IiifAv::DisplaysContent` in front of it?

Reply:
No.  It's a `Hyku::Application` class attribute.

Reply:
I've been moving too quickly between two branches.  I committed changes to reflect the correct state.